services/shared/src/main/java/com/wedriveu/services/shared/rabbitmq/VerticleConsumer.java

```diff
@@ -32,11 +33,13 @@ public VerticleConsumer(String name) {
     public void start() throws Exception {
         this.queueName = Constants.SERVICE_QUEUE_BASE_NAME + "." + name;
```

Comment:
so, is this not needed anymore? 
`baseRoutingKey = name.isEmpty() ? String.format(baseRoutingKey, name) : baseRoutingKey;`